packages/core/src/Core.ts

```diff
@@ -41,7 +41,7 @@ export class AgsCore {
    */
   public getConfigById(id: string): ServiceConfiguration | undefined {
     if (this.configurations) {
-      return find(this.configurations, service => service.id.toLowerCase() === id.toLowerCase());
+      return find(this.configurations, service => !!service.type && service.id.toLowerCase() === id.toLowerCase());
```

Comment:
why we are checking `service.type` here? Should it be `service.id && service.id.toLowerCase()`

Reply:
good catch @wei-lee copy and paste oversight :+1: